src/components/serverNotifications/serverNotifications.js

```diff
@@ -197,30 +197,30 @@ define(['connectionManager', 'playbackManager', 'events', 'inputManager', 'focus
         events.on(apiClient, "message", onMessageReceived);
     }
 
-    function enableNativeGamepadKeyMapping() {
-        if (window.navigator && "string" == typeof window.navigator.gamepadInputEmulation) {
-            window.navigator.gamepadInputEmulation = "keyboard";
-            return true;
+    function attachgamepad(e) {
+        if (navigator.getGamepads.length > 0) {
```

Comment:
```suggestion
        if (navigator.getGamepads().length > 0) {
```
?

Reply:
Thanks for noticing, wrote that ``if`` statement before the one in L216 and I changed the L216 one but forgot about the event handler. However, with that, you are receiving an array that contains 4 arrays, which are empty if there are no controllers attached. That means that the condition will always be true.

I'm still exploring ways in checking the number of gamepads connected, the proposed solution in Mozilla docs (the one I use in L216) doesn't convince me at all for our use case.

Will expand this information whenever I push the next commit to this PR.